src/components/actionSheet/index.js

```diff
@@ -188,3 +214,19 @@ export default class ActionSheet extends BaseComponent {
     );
   }
 }
+
+const styles = StyleSheet.create({
+  sheet: {
+    backgroundColor: Colors.white
+  },
+  dialog: {
+    backgroundColor: Colors.white
+  },
+  listWithTitle: {
+    paddingBottom: 8
+  },
+  listNoTitle: {
+    paddingTop: 8,
+    paddingBottom: 8
```

Comment:
Notice the duplication of paddingBottom. I like to pass one style to both cases (ex. action) and pass another one to the other case (ex. actionNoTitle). 
Also, it is better to move this value (8) to a const so it will be one source for the bunch.

Reply:
I get what you are saying, but in this case I feel like the code is more readable this way.
I'll add a const.

Reply:
My role in these situations is if I want to change the one value, would I need to change the other one as well? if so, I'm using a const. What is the case here?